2996_reflection/reflection.md

```diff
@@ -3122,6 +3156,31 @@ has its linkage determined as follows:
 
 :::
 
+Add a bullet to paragraph 13:
+
+::: std
+
+[13]{.pnum} A declaration `$D$` _names_ an entity `$E$` if
+
+* `$D$` contains a _lambda-expression_ whose closure type is `$E$`,
+* [`$D$` contains a _splice-specifier_ designating `$E$`,]{.addu}
+
+...
+
+:::
+
+Extend the definition of _TU-local_ values and objects to include reflections:
+
+::: std
+
+[16]{.pnum} A value or object is _TU-local_ if either
+
+* [16.1]{.pnum} it is, or is a pointer to, a TU-local function or the object associated with a TU-local variable, [or]{.rm}
+* [[16.2]{.pnum} it is a reflection representing a TU-local entity, or]{.addu}
+* [16.3]{.pnum} it is an object of class or array type and any of its subobjects or any of the objects or functions to which its non-static data members of reference type refer is TU-local and is usable in constant expressions.
```

Comment:
Keep the existing numbering (so the middle one becomes `16.1*` or something

Reply:
Done, used `16.1+` like you did above.